dashboard/src/app/projects/create-project/github/create-project-github.controller.js

```diff
@@ -179,6 +180,9 @@ export class CreateProjectGithubCtrl {
     this.selectedRepository = gitHubRepository;
     // broadcast event
     this.$rootScope.$broadcast('create-project-github:selected');
+    this.$timeout(() => {
+      this.repositorySelectNotify();
+    });
```

Comment:
I guess it would be better to send broadcast after calling method "repositorySelectNotify()". WDYT?